check-storage-usage/README.md

```diff
@@ -0,0 +1,52 @@
+---
+page_type: sample
+languages:
+  - csharp
+name: Check storage usage of Azure Cognitive Search
+description: "Demonstrates checking storage usage of an Azure Cognitive Search service. This example builds a C# Function App using the Azure Cognitive Search .NET SDK."
+products:
+  - azure
+  - azure-cognitive-search
+  - azure-functions
+urlFragment: check-storage-usage
+---
+
+# Check Azure Cognitive Search service storage usage
+
+![Flask sample MIT license badge](https://img.shields.io/badge/license-MIT-green.svg)
+
+Demonstrates checking storage usage of an Azure Cognitive Search service on a schedule. This sample may be modified to [adjust the service's capacity](https://docs.microsoft.com/azure/search/search-capacity-planning) or send an alert when the storage usage exceeds a predefined threshold.
+
+This .NET Core application runs as an [Azure Function](https://docs.microsoft.com/azure/azure-functions/functions-overview). The program [is deployed to Azure](https://docs.microsoft.com/azure/azure-functions/functions-create-your-first-function-visual-studio?tabs=in-process) using [Visual Studio](https://visualstudio.microsoft.com/downloads/) and [runs automatically on a predefined schedule](https://docs.microsoft.com/azure/azure-functions/functions-create-scheduled-function).
+
+## Prerequisites
+
+- [Visual Studio](https://visualstudio.microsoft.com/downloads/)
+- [Azure Cognitive Search service](https://docs.microsoft.com/azure/search/search-create-service-portal)
+- [Azure Functions](https://docs.microsoft.com/azure/azure-functions/functions-overview)
+
+## Setup
+
+1. Clone or download this sample repository.
+
+1. Extract contents if the download is a zip file. Make sure the files are read-write.
+
+## Run the sample
+
+1. Deploy the sample to Azure [using Visual Studio](https://docs.microsoft.com/azure/azure-functions/functions-create-your-first-function-visual-studio?tabs=in-process#publish-the-project-to-azure).
+
+1. Navigate to the deployed Function App in the Azure Portal.
+
+1. [Update the application settings of the Function App](https://docs.microsoft.com/azure/azure-functions/functions-how-to-use-azure-function-app-settings?tabs=portal). In the Azure portal, navigate to **Configuration** section under **Settings**. Add the following **Application Settings**:
```

Comment:
I'd recommend checking in the local.settings.json file just so customers don't have to figure out how to add it themselves. We've typically included that file in the powerskils repo: https://github.com/Azure-Samples/azure-search-power-skills/tree/main/Template/HelloWorld

Along those same lines, It might also be worth adding here instructions on how to run it locally (customers will need to run it locally so they can edit it and add the code to send the notification).

Reply:
I'll link to the documentation that already exists here. Checked in local settings file